pipeline/metadata/satellite.py

```diff
@@ -179,8 +243,8 @@ def _add_vantage_point_tags(
 
   # PCollection[Tuple[Tuple[date,ip],Dict[input_name_key,List[Row]]]]
   grouped_metadata_and_rows = (({
-      IP_METADATA_PCOLLECTION_NAME: ips_with_metadata,
-      ROWS_PCOLLECION_NAME: rows_keyed_by_ip_and_date
+      BEAM_COGROUP_A_SIDE: ips_with_metadata,
```

Comment:
Use meaningful names that represent the data.

Reply:
done